charts/retool/templates/deployment_jobs.yaml

```diff
@@ -58,6 +58,10 @@ spec:
 {{ toYaml .Values.commandline.args | indent 10 }}
         {{- end }}
         env:
+          - name: DEPLOYMENT_TEMPLATE_TYPE
+            value: {{ template "retool.deploymentTemplateType" . }}
+          - name: DEPLOYMENT_TEMPLATE_VERSION
+            value: {{ template "retool.deploymentTemplateVersion" . }}
```

Comment:
really only needed here, but in case we change the analytics implementation on boot.

(and just including code executor here as well)